behaviors/wow_retail/druid/restoration.lua

```diff
@@ -164,23 +189,33 @@ local function DruidRestoHeal()
     local u = v.Unit
     local prio = v.Priority
 
-    if wildgrowth and Spell.WildGrowth:CastEx(u) then return end
 
+    -- TODO convoke and tranq logic need to take into account Multiple people low
+    if Settings.DruidRestoOvergrowth and u.HealthPct < 25 and Spell.Overgrowth:CastEx(u) then return end
     if u.HealthPct < 50 and Spell.CenarionWard:CastEx(u) then return end
     if u.HealthPct < 50 and (u:HasBuffByMe("Rejuvenation") or u:HasBuffByMe("Regrowth")) and
-        Spell.Swiftmend:CastEx(u, SpellCastExFlags.NoUsable) then return end
+        Spell.Swiftmend:CastEx(u, SpellCastExFlags.NoUsable) then
+      return
+    end
+
+    if Settings.DruidInnervate and Me.PowerPct < 25 and Spell.Innervate:CastEx(Me) then return end

```

Comment:
Innervate should only be used in high healing situations because it does not regenerate mana, it makes all spells free